packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -45,18 +43,52 @@ const useStyles = makeStyles(theme => ({
   },
   summary: {
     '&.Mui-expanded': {
-      borderBottom: `1px solid ${theme.palette.divider}`
+      borderBottom: `1px solid ${theme.palette.divider}`,
+
+      '& > .MuiAccordionSummary-content': {
+        margin: '12px 0px'
+      }
     }
   },
   accordionHeader: {
     lineHeight: 2.5
+  },
+  indexText: {
+    marginRight: 12
+  },
+  headerContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '100%',
+    alignItems: 'center'
+  },
+  parametersItem: {
+    display: 'flex',
+    width: '100%',
+    padding: '0px 16px 16px 16px'
+  },
+  itemContainer: {
+    width: '33.33%',
+    display: 'flex',
+    flexDirection: 'column',
+    maxHeight: 200,
+    marginLeft: 8,
+    marginRight: 8
+  },
+  wrappedText: {
+    overflow: 'auto',
+    overflowWrap: 'anywhere'
+  },
+  boldText: {
+    fontWeight: 'bold'
```

Comment:
1. for spacing (e.g. margin, padding), use the spacing unit from the theme
2. for sizes, use flex sizes, avoid percentages
3. for fonts, use typography

Reply:
On it

Reply:
Done